app/src/main/java/com/infomaniak/mail/data/api/ApiRepository.kt

```diff
@@ -334,8 +333,9 @@ object ApiRepository : ApiRepositoryCore() {
         "output" to "mail",
     )
 
-    fun checkFeatureFlag(featureFlagType: FeatureFlagType): ApiResponse<Map<String, Boolean>> {
-        return callApi(ApiRoutes.featureFlag(featureFlagType.apiName), GET)
+    fun checkFeatureFlag(featureFlag: FeatureFlag, currentMailboxUuid: String): ApiResponse<Map<String, Boolean>> {
+        val body = mapOf("mailbox_uuid" to currentMailboxUuid)
```

Comment:
Useless body for a GET request

Reply:
I changed it to be passed as a parameter as this is what was required